main.go

```diff
@@ -460,22 +460,31 @@ func addExtensions(msg *descriptor.DescriptorProto, extensions []*descriptor.Fie
 			continue
 		}
 
+		if len(extendeeNodes) < 1 {
```

Comment:
Actually, my bad, that should be 

```suggestion
		if len(extendeeNodes) < 2 {
```

as we'll have one element even without `.` or an empty string. As soon as there would be a `.` we're guaranteed to have at least 2 elements.